File: pdoo/java/Deepspace/BetaPowerEfficientSpaceStationToUI.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Deepspace;

/**
 *
 * @author Profe
 */
public class BetaPowerEfficientSpaceStationToUI extends PowerEfficientSpaceStationToUI {

    BetaPowerEfficientSpaceStationToUI(BetaPowerEfficientSpaceStation s) {
        super(s);
    }
    
  @Override
  public String getName () {
    return super.getName() + " (beta)";
  }
    
}

File: pdoo/ruby/lib/BetaPowerEfficientSpaceStationToUI.rb

require_relative 'PowerEfficientSpaceStationToUI'

module Deepspace

# 24.5.17 - Translation from Java
# @author Profe
  
class BetaPowerEfficientSpaceStationToUI < PowerEfficientSpaceStationToUI

    def initialize (s)
        super(s)
    end
    
    #Override
    def name
      return super + " (beta)"
    end

end # class

end # module
